ts/Extensions/Annotations/Annotations.ts

```diff
@@ -1342,17 +1342,6 @@ merge<Annotation>(
                  * @apioption annotations.labels.point.y
                  */
 
-                /**
-                 * This number defines which xAxis the point is connected to.
-                 * It refers to either the axis id or the index of the axis in
-                 * the xAxis array. If the option is not configured or the axis
-                 * is not found the point's x coordinate refers to the chart
-                 * pixels.
-                 *
-                 * @type      {number|string|null}
-                 * @apioption annotations.labels.point.xAxis
```

Comment:
`labels[index].point.xAxis` is supported: https://api.highcharts.com/highstock/annotations.labels.point.xAxis